src/main/java/com/amihaiemil/versioneye/Me.java

```diff
@@ -57,4 +59,10 @@
      * @return Favorites
      */
     Favorites favorites();
+    
+    /**
+     * Post authenticated user to the server.
+     * @param authenticated Authenticated user.
+     */
+    void post(JsonObject authenticated);
```

Comment:
@amihaiemil So I should remove the post() method and ``MkVersionEye`` must take ``MkUser`` in its ctor ?
Also for testing should I skip it now and add it to same puzzle :D ?